src/Utilities/Routes.js

```diff
@@ -68,27 +32,21 @@ InsightsRoute.propTypes = {
     rootClass: PropTypes.string
 };
 
-/**
- * the Switch component changes routes depending on the path.
- *
- * Route properties:
- *      exact - path must match exactly,
- *      path - https://prod.foo.redhat.com:1337/insights/advisor/rules
- *      component - component to be rendered when a route has been chosen.
- */
 export const Routes = () => {
-
+    let location = useLocation();
+    let path = location.pathname;
     return (
         <Switch>
-            <InsightsRoute path={'/:subpage?'} component={Vulnerabilities} rootClass="Vulnerabilities" />
+            <Redirect from={PATHS.cvepagePath.to} to={PATHS.inventoryDetail.to} />
+            <InsightsRoute exact path={PATHS.cvePage.to} component={CVEPage} />
+            <InsightsRoute exact path={PATHS.inventoryDetail.to}  component={InventoryDetail} />
+            <InsightsRoute path={PATHS.vulnerabilitiesCves.to} component={VulnerabilitiesCVEs} />
```

Comment:
Everything are neat as usual, I did not see any issues. Just for my curiosity, there is one quetion. I think that you intentionally did not add exact prop here to make this route as default for 'cves'. Do you think the the same applies to Systems component route?

Reply:
good point. Basically no I didn't do it intentionally. I did intentionally though the order cause If you have similar paths like `cves` and `cves/:cveID` without the `exact` the order matters. 

For example, having this order, the cve detail page will never render
`            <InsightsRoute path={PATHS.vulnerabilitiesCves.to} component={VulnerabilitiesCVEs} />
`
`            <InsightsRoute exact path={PATHS.cvePage.to} component={CVEPage} />
`

Anyways, I think it's a good point so I will add the exact to vulnerabilitiesCves as well :D